src/pages/UnLikedFoodPage.js

```diff
@@ -129,6 +133,12 @@ const Header = styled.header`
   }
 `;
 
+const HeadingStyled = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-right: 1rem;
```

Comment:
This is different than in your LikedFoodPage.js, is it intentional?

Reply:
yes, the integration of the component for sorting at the LikedFoodPage is a little bit different for that of the component searchbar at the UnLikedFoodPage